.github/workflows/qc_checks.yaml

```diff
@@ -57,8 +57,10 @@ jobs:
           python3 check_js_templates.py
       - name: Lint Javascript Files
         run: |
-          invoke render-js-files
-          npx eslint js_tmp/*.js
+          python InvenTree/manage.py prerender
+          echo "Checking translated javascript files:"
+          ls InvenTree/InvenTree/static_i18n/i18n/
```

Comment:
Question: is this still needed?

Reply:
Otherwise where are the javascript files linted?

Reply:
@schroedingersgat but what does line 61 do in that respect? Maybe I am overlooking something but that just fills the logs with a directory  content list.